addons/medical_damage/functions/fnc_woundsHandlerSQF.sqf

```diff
@@ -122,8 +122,10 @@ private _bodyPartVisParams = [_unit, false, false, false, false]; // params arra
                 _critialDamage = true;
             };
             if ([_unit, _bodyPartNToAdd, _bodyPartDamage, _woundDamage] call FUNC(determineIfFatal)) then {
-                TRACE_1("determineIfFatal returned true",_woundDamage);
-                [QEGVAR(medical,FatalInjury), _unit] call CBA_fnc_localEvent;
+                if !(isPlayer _unit && {random 1 > EGVAR(medical,deathChance)}) then {
```

Comment:
`true && true -> !true -> false`, so if player and death chance above, it won't cause fatal injury? This seems reversed.

Reply:
De Morgan's law, the code as is can be simplified to:

!isPlayer || random 1 <= deathChance

Reply:
You are correct, I was still reading it as survivability chance. If death chance is 0.15 then random value has to be below that, not above, so you get `true && false` when it should happen actually. :D

I would prefer inverting that with De Morgan's law as you state, easier to read. But that may just be me still stuck with "survival chance" making more sense in code, but probably not to the user.

Reply:
@BrettMayson Can we reverse that? Then I am happy to merge.

Reply:
Yeah sure I can invert it